cmd/minishift/cmd/start_preflight.go

```diff
@@ -0,0 +1,278 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
used `start.go` as boilerplate. should have used something else...

changed now